lib/checks/tables/td-headers-attr.js

```diff
@@ -43,20 +39,22 @@ var badCells = cells.reduce(function(badCells, cell) {
 		}
 
 		// Check if the headers are of cells inside the table
-		notOfTable = headers.reduce(function(fail, header) {
-			return fail || ids.indexOf(header) === -1;
-		}, false);
+		notOfTable = headers.find(header => !ids.includes(header));
```

Comment:
Might be better to use `some` instead of `find` to make notOfTable a boolean value. I misread it because of this the first time around.

```suggestion
		notOfTable = headers.some(header => !ids.includes(header));
```